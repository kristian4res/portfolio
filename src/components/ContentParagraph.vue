<template>
  <p class="text-lg sm:text-wrap my-2">
    <slot></slot>
  </p>
</template>
