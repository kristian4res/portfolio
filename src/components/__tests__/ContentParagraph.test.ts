import { mount } from '@vue/test-utils'
import { describe, it, expect } from 'vitest'
import ContentParagraph from '../ContentParagraph.vue'

describe('ContentParagraph.vue', () => {
  it('renders correctly and matches snapshot', () => {
    const wrapper = mount(ContentParagraph, {
      slots: {
        default: 'Hello, world! This is a test.',
      },
    })
    expect(wrapper.html()).toMatchSnapshot()
  })
})
