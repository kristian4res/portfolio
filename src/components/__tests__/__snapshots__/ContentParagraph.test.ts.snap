// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`ContentParagraph.vue > renders correctly and matches snapshot 1`] = `"<p class="text-lg sm:text-wrap my-2">Hello, world! This is a test.</p>"`;
